docs/pages/connect-your-client/tsh.mdx

```diff
@@ -6,142 +6,244 @@ labels:
-$ tsh ssh -o ForwardAgent=yes user@node
-$ tsh ssh -o AddKeysToAgent=yes user@node
+### List resources that you can access.
 
-# You can even create a convenient symlink:
-$ ln -s /path/to/tsh /path/to/ssh
+In a Teleport cluster, all Teleport Agents periodically ping the cluster's Auth
+Service and update their status. This allows Teleport users to see which
+Teleport-protected resources are online.
 
-# ... and now your 'ssh' command is calling Teleport's `tsh ssh`
-$ ssh user@host
+This command lists all connected servers in the cluster that you have permission
+to access:
 
-# This command removes SSH certificates from a user's machine:
-$ tsh logout
+```code
+$ tsh ls
+
+# Node Name     Address            Labels
+# ---------     -------            ------
+# turing        ⟵ Tunnel          os=linux
+# graviton      10.1.0.7:3022      os=osx
 ```
 
-</TabItem>
-<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
+You can use `tsh` commands to list other kinds of resources. For more
+information, see the `tsh` reference entries for the following resource types:
+
+|Resource|Command|
+|---|---|
+|[Applications](../reference/cli/tsh.mdx#tsh-apps-ls)|`tsh apps ls`|
+|[Databases](../reference/agent-services/database-access-reference/cli.mdx#tsh-db-ls)|`tsh db ls`|
+|[Kubernetes clusters](../reference/agent-services/database-access-reference/cli.mdx#tsh-kube-ls)|`tsh kube ls`|
+|[Servers](../reference/cli/tsh.mdx#tsh-ls)|`tsh ls`|
+
+Windows desktops are available to list in Teleport Connect and the Teleport Web
+UI.
+
+`tsh <resource> ls` commands can apply a filter based on the resource's
+labels.  For example, to only show servers with the `os` label set to `osx`,
+you can run the following command:
 
 ```code
-# Login into a Teleport cluster. This command retrieves the user's certificates
-# and saves them into ~/.tsh/mytenant.teleport.sh
-$ tsh login --proxy=mytenant.teleport.sh
+$ tsh ls os=osx
+
+# Nodename      Address            Labels
+# ---------     -------            ------
+# graviton      10.1.0.7:3022      os=osx
+```
+
+<details>
+<summary>Not seeing resources?</summary>
+
+(!docs/pages/includes/node-logins.mdx!)
 
-# SSH into a Node as usual
-$ tsh ssh user@node
+</details>
 
-# `tsh ssh` takes the same arguments as the OpenSSH client:
-$ tsh ssh -o ForwardAgent=yes user@node
-$ tsh ssh -o AddKeysToAgent=yes user@node
+### Connect to a resource
 
-# You can even create a convenient symlink:
-$ ln -s /path/to/tsh /path/to/ssh
+Once you have determined a resource to connect to, the next step is to access
+the resource with `tsh`, which handles authentication to your Teleport cluster
+and routing traffic to and from local clients.
 
-# ... and now your 'ssh' command is calling Teleport's `tsh ssh`
-$ ssh user@host
+You can only connect to a Windows desktop using the Teleport Web UI or Teleport
+Connect. To do so, use the left sidebar to navigate to **Resources > Desktops**
+and select a desktop to connect to.
```

Comment:
Connect doesn't have a sidebar, I guess we could either reword this to say "To do so in the Web UI" or remove the second sentence completely?